File: Basic Java/TestingSystem_Assignment/src/com/vti/backend/TestingSystem_Assignment_4/KySu.java
package com.vti.backend.TestingSystem_Assignment_4;

public class KySu extends CanBo{
    private String industry;

    public KySu(long id, String fullName, int age, Gender gender, String address, String industry) {
        super(id, fullName, age, gender, address);
        this.industry = industry;
    }
}

File: ReactJS/Assignment_02/Lab_02/KySu.ts
import {CanBo, Gender} from './CanBo';


export class KySu extends CanBo {
    private training: string;


	constructor(fullName: string, age: number, gender: Gender, address: string, training: string) {
        super(fullName, age, gender, address);
		this.training = training;
	}



    /**
     * Getter training
     * @return {string}
     */
	public getTraining(): string {
		return this.training;
	}

    /**
     * Setter training
     * @param {string} value
     */
	public setTraining(value: string) {
		this.training = value;
	}


    public displayDetails(){
        super.displayDetails();
        console.log( `Tranining: ${this.getTraining()}`);
    }
    
}
